hl7apy/mllp.py

```diff
@@ -47,50 +54,54 @@ def __str__(self):
 
 
 class _MLLPRequestHandler(StreamRequestHandler):
+    encoding = 'utf-8'
```

Comment:
UTF-8 seems to be the most common encoding in the US. From what I've read, LATIN-1 is the most common for HL7 in Europe. I don't have an immediate need for this module, so I'm happy to set it to whatever encoding you'd prefer.

`ascii` would be the most restrictive, and would work on the common subset of UTF-8 and LATIN-1, so might be a good compromise.